public/scripts/app.js

```diff
@@ -26,17 +26,17 @@ $(document).ready(function(){
   // make a GET request for all restaurants
   $.ajax({
     method: 'GET',
-    url: '/api/restaurant',
+    url: '/api/restaurants',
     success: handleSuccess,
     error: handleError
   });
 
-  $('#singlebutton').on('submit', function(e) {
+  $('#restaurant-form form').on('submit', function(e) {
```

Comment:
You need to attach the `.on('submit'` handler to a form, not a button. Forms are submitted, buttons are clicked.